app/models/user.rb

```diff
@@ -228,6 +228,15 @@ def post_successful_login
     User.current = self
   end
 
+  def set_locations_and_organizations_from_usergroups(usergroups)
```

Comment:
This should probably be private, the method doesn't get called from anywhere else.